app/controllers/application_controller.rb

```diff
@@ -5,6 +5,133 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
+  end
+get '/tweets/new' do 
+    if Helpers.is_logged_in?(session)
+      erb :'/tweets/create_tweet'
+    else 
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect :'/tweets/new'
+    else
+      user = User.find_by_id(session[:user_id])
+      Tweet.create(content: params[:content], user_id: user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.